common/middleware/ensure-authenticated.js

```diff
@@ -9,13 +9,33 @@ function _isExpired(authExpiry) {
 module.exports = function ensureAuthenticated({
   provider,
   whitelist = [],
+  expiryMargin = 5 * 60,
```

Comment:
Do we need to set the default in two places? That could be confusing if we want to update it.

Maybe just set the default here?

Reply:
I think we need the ability to change it without having to make a code commit - hence the env var. I just like having a fallback value in the middleware params (especially since we're not testing what we're sending through to the middleware instantiation from `app/server`)

Reply:
Yep, totally happy with having an env var to be able to change it per environment. I just meant we're setting a default in the config file **and** here. Maybe if we want a default we should set it in one place - here?

Reply:
Defaulting it to undefined here and skipping if so